components/other-utilities/other-utilities.css

```diff
@@ -0,0 +1,65 @@
+.toast {
+  width: 20rem;
+  padding: 1rem;
+  background-color: var(--dark-blue);
+  color: var(--white);
+  border-bottom: 8px solid var(--primary);
+  text-align: center;
+}
+
+.star {
+  font-size: 1.5em;
+  color: var(--primary);
+}
+
+.slider-container {
+  min-width: 12.5rem;
+  max-width: max-content;
+}
+
+.slider {
+  -webkit-appearance: none;
+  width: 100%;
+  height: 1.5rem;
+  background: var(--secondary-lighter);
+  outline: none;
+  opacity: 0.7;
+  -webkit-transition: 0.2s;
+  transition: opacity 0.2s;
+}
+.slider:hover {
+  opacity: 1;
+}
+
+.slider::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  background: var(--primary-dark);
+  cursor: pointer;
+  width: 1.25rem;
+  height: 2.25rem;
+}
+
+.slider::-moz-range-thumb {
+  background: var(--primary-dark);
+  cursor: pointer;
+}
+
+.price-datalist {
+  display: flex;
+  justify-content: space-between;
```

Comment:
use classes you made in main.css file here

Reply:
yes, makes sense. Fixing it